src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -23,6 +23,9 @@
     ];
     const form = doc.querySelector('.ez-form-validate');
     const submitBtns = form.querySelectorAll('[type="submit"]:not([formnovalidate])');
+    const menuWorkflowButtons = doc.querySelectorAll('button[id^="workflow__apply__"]');
```

Comment:
Still the same issue, workflow code cannot be in admin-ui

Reply:
I working on this, I change this on WiP